components/mautic/actions/create-contacts/create-contacts.mjs

```diff
@@ -196,6 +196,11 @@ export default {
       description: "If true, then empty values are set to fields. Otherwise empty values are skipped",
       optional: true,
     },
+    tags: {
+      type: "string[]",
+      description: "Tags that will be assigned to new contact.",
+      optional: true
+    }
```

Comment:
Actually I missed this eslint error, fix it please!
```suggestion
      optional: true,
    },
```

Reply:
@jcortes pushed the fix